Public/Src/FrontEnd/MsBuild/MsBuildWorkspaceResolver.cs

```diff
@@ -277,11 +277,11 @@ private BuildParameters.IBuildParameters RetrieveBuildParameters()
             return buildParameters;
         }
 
-        private bool TryRetrieveParsingEntryPoint(out AbsolutePath parsingEntryPoint)
+        private bool TryRetrieveParsingEntryPoint(out IEnumerable<AbsolutePath> parsingEntryPoints)
         {
-            if (m_resolverSettings.FileNameEntryPoint.HasValue)
+            if (m_resolverSettings.FileNameEntryPoints != null && m_resolverSettings.FileNameEntryPoints.Count > 0)

```

Comment:
Could just use elvis operator: `m_resolverSettings.FileNameEntryPoints?.Count`

Reply:
It seems much more declarative to do the check

Reply:
+1 to Danny's comment